src/runtime_src/xdp/profile/device/profile_ip_access.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2019-2020, Xilinx Inc - All rights reserved
+ * Xilinx Debug & Profile (XDP) APIs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "profile_ip_access.h"
+
+namespace xdp {
+
+ProfileIP::ProfileIP(xclDeviceHandle handle, int index) {
+    // initialize member variables

```

Comment:
Use constructor initializer list.

Reply:
Changed to suggested approach in the new push.